kubernetes/docs/kind-deployment.md

```diff
@@ -28,14 +28,13 @@ export GAFFER_TOOLS_VERSION=${GAFFER_TOOLS_VERSION:-2.0.0-alpha-0.3}
 
 docker-compose --project-directory ../docker/accumulo/ -f ../docker/accumulo/docker-compose.yaml build
 docker-compose --project-directory ../docker/gaffer-ui -f ../docker/gaffer-ui/docker-compose.yaml build
-docker-compose --project-directory ../docker/gaffer-operation-runner/ -f ../docker/gaffer-operation-runner/docker-compose.yaml build
+docker-compose --project-directory ../docker/gaffer-road-traffic-loader/ -f ../docker/gaffer-road-traffic-loader/docker-compose.yaml build
```

Comment:
Am I right to think road-traffic should have been here before anyway?

Reply:
Not quite, the `gaffer-operation-runner` compose file also built the `gaffer-road-traffic-loader`, `gaffer-rest` and `gaffer` images. Because that is now gone, I am building the `gaffer-road-traffic-loader` compose file to build the `gaffer-rest` and `gaffer` images.